cmd/seccomp-operator/main.go

```diff
@@ -30,10 +30,9 @@ import (
 )
 
 const (
-	appName                 string = "seccomp-operator"
-	namespaceToWatchDefault string = appName
-	namespaceToWatchKey     string = "NAMESPACE_TO_WATCH"
-	jsonFlag                string = "json"
+	appName             string = "seccomp-operator"
+	namespaceToWatchKey string = "NAMESPACE_TO_WATCH"
```

Comment:
Do we wanna remove this completely?

Reply:
Yeah, good question. I just left it intact for now because I couldn't answer that myself ;)

Reply:
Removed it. :+1: